test/ui-test/src/screens/StatusChatScreen.py

```diff
@@ -57,16 +62,24 @@ def send_message(self, message: str):
         
     # Verifications region:        
     def verify_last_message_sent(self, message: str):
-        [loaded, last_message_obj] = is_loaded_visible_and_enabled(ChatComponents.LAST_MESSAGE_TEXT.value)
-        verify(loaded, "Checking last message sent: " + message)
-        verify_text_contains(str(last_message_obj.text), str(message))
+        # Get the message text
+        # We don't search for StatusTextMessage_chatText directly, because there're 2 instances of it in a reply message
+        last_message_obj = self.get_message_at_index(0)
+        text_message_obj = getChildrenWithObjectName(last_message_obj, "StatusMessage_textMessage")[0]
+        text_edit_obj = getChildrenWithObjectName(text_message_obj, "StatusTextMessage_chatText")[0]
```

Comment:
It,s better to put those names in an enum like the others at the top

Reply:
If you prefer, you can create another `Class` for just constant names then.

Reply:
I agree it is better to add them into an enum. A specific class for constants is fine!!